src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -14,107 +19,109 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.LinkedHashMap;
-import java.util.List;
 import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
 
 public class KafkaMetricsCollectorTest {
 
     private final MetricConfig metricConfig = new MetricConfig();
     private final Time time = Time.SYSTEM;
-    private Map<String, String> labels;
+    private Map<String, String> tagsMap;
+    private Labels labels;
 
     @BeforeEach
     public void setup() {
-        labels = new HashMap<>();
-        labels.put("key", "value");
+        tagsMap = new LinkedHashMap<>();
```

Comment:
The setup of the 2 Collector tests is still a bit wonky with the `labels`/`tagsMap` fields. I'll see if I can tidy this up a little bit.